api/src/pcapi/routes/backoffice_v3/tags.py

```diff
@@ -24,8 +27,26 @@
 
 @tags_blueprint.route("", methods=["GET"])
 def list_tags() -> utils.BackofficeResponse:
-    tags = criteria_models.Criterion.query.all()
-    return render_template("tags/list_tags.html", rows=tags)
+    form = SearchTagForm(request.args)
+
+    if not form.validate() or form.q.data is None:
```

Comment:
Est-ce qu'utiliser directement un `PCSearchField` au lieu d'un `PCOptSearchField` dans `SearchTagForm` ne permettrait pas de supprimer la partie `or form.q.data is None` ?